### PR TITLE
Sets honorLabels default value to true. Fixes #53

### LINKED ISSUE #53: Non-working default in ServiceMonitor value
When `honorLabels` is set to `false`, OpenCost metrics are being ingested with the `namespace` label set to the opencost namespace. This causes most of the data to be set to `unallocated`, as the ingested namespace labels do not match the pods.

### PR DESCRIPTION
**Pull request by edmeister**

### CODE REVIEW

**Comment by mattray** [comment]
I see in [Kubecost this defaults to "true"](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/values.yaml#L531), so I think I'm good with this. Could you update the README.md to document the change in the default?

**Comment by edmeister** [comment]
> 

Done

File: charts/opencost/Chart.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ keywords:
   - kubecost
   - opencost
   - monitoring
-version: 1.10.0
+version: 1.11.0
```
Comment (toscott) [inline charts/opencost/Chart.yaml:12]: @edmeister - This version still needs bumped to 1.12.0 for CI checks to pass. Doesn't look like I can edit your branch or I would. Sorry!

**Comment by toscott** [comment]
Will pull into a bulk merge and update version.

**Comment by toscott** [comment]
Merging here:
https://github.com/opencost/opencost-helm-chart/pull/60

**Pull request closed without merging**